charts/catalog/Chart.yaml

```diff
@@ -1,3 +1,3 @@
 name: catalog
 description: service-catalog API server and controller-manager helm chart
-version: 0.1.42
+version: 0.1.43
```

Comment:
Don't do this.
@jberkhahn will change this on next release.

Reply:
yeah, this is a value that is incremented when we cut releases so the helm chart points at the right image. This will break the helm chart if you do it before we release 0.1.43

Reply:
@jberkhahn Do you want I set the version to 0.1.42 again ?
I do want you want about that

Reply:
Yeah, just reset it

Reply:
I reset it manually ;-)

Reply:
/assign @jberkhahn